README.md

```diff
@@ -205,6 +209,33 @@ const toast = document.createElement('std-toast');
 console.log(toast.open); // false
 ```
 
+##### `closeButton` property
+
+The `closeButton` property allows controlling the element's `closebutton=""` attribute.
+It is similar to the reflected properties,
+but slightly more complicated to allow both a boolean usage model
+(for using the default close button content)
```

Comment:
Should we mention what the default content is?

Reply:
I was thinking it'd be up to the UA.